docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/is-currently-used-php-version-8-0.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Check if the PHP version used is 8.0 or higher
+description: This document allows you to assess if a project uses PHP Version >=8.0.
+template: howto-guide-template
+---
+
+
+
+## Resources for assessment 
+
+Backend
+
+## Description
+
+In this assessment step, you need to check if the project is compatible with PHP 8.0.
```

Comment:
What if the project is already using PHP 8.0? And how do I identify it?

Reply:
It can be identified using PHPCompatibility or Rector package. If a tool says there are no incompatibilities with PHP 8.0 then there is nothing to do.